homeassistant/components/binary_sensor/bmw_connected_drive.py

```diff
@@ -0,0 +1,123 @@
+"""
+Reads vehicle status from BMW connected drive portal.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.bmw_connected_drive/
+"""
+import logging
+import asyncio
```

Comment:
Please sort 🔤 within groups standard library, 3rd party and homeassistant.

Reply:
Done